cmd/skaffold/skaffold.go

```diff
@@ -19,20 +19,38 @@ package main
 import (
 	"context"
 	"errors"
+	"log"
 	"os"
 
+	"cloud.google.com/go/profiler"
 	"github.com/sirupsen/logrus"
 
 	"github.com/GoogleContainerTools/skaffold/cmd/skaffold/app"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/instrumentation"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/output"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/version"
 )
 
 type ExitCoder interface {
 	ExitCode() int
 }
 
 func main() {
+	if _, ok := os.LookupEnv("SKAFFOLD_PROFILE"); ok {
```

Comment:
```suggestion
	if _, ok := os.LookupEnv("SKAFFOLD_PROFILER"); ok {
```

See here https://skaffold.dev/docs/references/cli/#skaffold-apply ( search for `SKAFFOLD_PROFILE`)

Reply:
Changed s/SKAFFOLD_PROFILE/SKAFFOLD_PROFILER